.travis.yml

```diff
@@ -22,3 +22,5 @@ matrix:
     - rvm: "2.3.0"
     - rvm: "2.3.3"
     - rvm: "2.4.1"
+    - rvm: "2.5.7"
+    - rvm: "2.6.5"
```

Comment:
What about changing the gemspec file and set the minimum supported ruby version?

Reply:
I was thinking about this too... Do you think that we can already remove support for versions that reached their EOL?

Reply:
Humm, I think it is not a good idea since I know some apps that are using these old ruby versions